### ISSUE TITLE
Test failure under Python 3.7

### ISSUE THREAD

**Issue by felixonmars** [PyFilesystem/pyfilesystem2#185]
```
======================================================================
ERROR: test_entry_point_type_error (tests.test_opener.TestRegistry)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/build/python-fs/src/pyfilesystem2-2.0.24/tests/test_opener.py", line 213, in test_entry_point_type_error
    opener.open_fs('test://')
  File "/build/python-fs/src/pyfilesystem2-2.0.24/fs/opener/registry.py", line 192, in open_fs
    default_protocol=default_protocol
  File "/build/python-fs/src/pyfilesystem2-2.0.24/fs/opener/registry.py", line 147, in open
    opener = self.get_opener(protocol)
  File "/build/python-fs/src/pyfilesystem2-2.0.24/fs/opener/registry.py", line 97, in get_opener
    if not isinstance(opener, object) or not issubclass(opener, Opener):
  File "/usr/lib/python3.7/abc.py", line 143, in __subclasscheck__
    return _abc_subclasscheck(cls, subclass)
TypeError: issubclass() arg 1 must be a class

----------------------------------------------------------------------
Ran 1350 tests in 7.726s
```

**Comment by althonos** [comment]
@willmcgugan I'm on it.

**Comment by felixonmars** [comment]
Looks like the issue is still present, PR #186 didn't test for 3.7 :(

I tested it with the newly released 2.0.25 release with the commit included, but still getting the same error:

```
======================================================================
ERROR: test_entry_point_type_error (tests.test_opener.TestRegistry)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/build/python-fs/src/pyfilesystem2-2.0.25/tests/test_opener.py", line 213, in test_entry_point_type_error
    opener.open_fs('test://')
  File "/build/python-fs/src/pyfilesystem2-2.0.25/fs/opener/registry.py", line 192, in open_fs
    default_protocol=default_protocol
  File "/build/python-fs/src/pyfilesystem2-2.0.25/fs/opener/registry.py", line 147, in open
    opener = self.get_opener(protocol)
  File "/build/python-fs/src/pyfilesystem2-2.0.25/fs/opener/registry.py", line 97, in get_opener
    if not issubclass(opener, Opener):
  File "/usr/lib/python3.7/abc.py", line 143, in __subclasscheck__
    return _abc_subclasscheck(cls, subclass)
TypeError: issubclass() arg 1 must be a class

----------------------------------------------------------------------
Ran 1512 tests in 24.422s

FAILED (SKIP=25, errors=1)
```

**Comment by althonos** [comment]
@felixonmars : see #198 

Unfortunately Travis-CI does not support Python 3.7 yet, because of OpenSSL version conflict, so we can't test (except if doing all our tests in a docker image, which is cumbersome). But it should work with my fix now.

**Comment by willmcgugan** [comment]
Thanks @althonos Confirmed that fixes it when I run the tests locally.

@felixonmars Will have a new release soon.

**Comment by willmcgugan** [comment]
Fixed in 2.0.26
